src/logic/schedule-parser/extra-workers-info.parser.ts

```diff
@@ -46,7 +47,7 @@ export class ExtraWorkersInfoParser implements ExtraWorkersInfoProvider {
 
     if (slicedChildrenRow.length !== this.metaData.dayCount) {
       this.logLoadFileError(
-        "Sekcja pracownicy dzienni nie ma oczekiwanych wymiarów. Dla brakujących dni przyjęto, że liczba pracowników dziennych wynosi 0"
+        "Sekcja pracownicy dzienni nie ma oczekiwanych wymiarów. Dla brakujących dni przyjęto, że liczba pracowników dziennych wynosi 0."
```

Comment:
o co chodziło Adze z tym używaniem słowa _przyjęto_? można jednak zostawić?

Reply:
Samo _przyjęto, że_ nie przekazuje dobrze wiadomości o tym, co właściwie się zadziało i jak można to zmienić. Użytkownik potrzebuje informacji o tym, co jest inaczej niż mógł chcieć i co musi zrobić, żeby to naprawić. Moim zdaniem w tym przypadku wie, jak zmienić liczbę pracowników? Dlatego tak zostawiłam.
Ale pomyślę jeszcze nad tym.